doc/binning_basics.rst

```diff
@@ -0,0 +1,102 @@
+Using the binning module
+========================
+
+Stile contains some simple functions to bin your data.  Two of them--:class:`stile.BinStep
+<stile.binning.BinStep>` and :class:`stile.BinList <stile.binning.BinList>`--have simple,
+predefined ways of acting on your data set; the third, :class:`stile.BinFunction
+<stile.binning.BinFunction>`, uses user-defined functions to split your data.
+
+Basic interface
+---------------
+
+To start binning your data, you create a :class:`Bin*` object that will contain a binning
+definitions.  For instance, if you wanted to bin the ``ra`` column in 10 bins:
+
+>>> bin_object = stile.BinStep(field='ra', n_bins=10, low=0, high=360)
+
+To use it, call the object; it returns a list of objects which you can apply to your data to
+produce properly binned subsets.
+
+>>> for single_bin in bin_object():
+>>>     binned_data = single_bin(data)
```

Comment:
Perhaps clarify what ```binned_data``` is; is it a subset of the data in the same format as ```data``` itself, does the format change at all, ...?